pydocstringformatter/formatting/base.py

```diff
@@ -0,0 +1,27 @@
+import abc
+import tokenize
+
+
+class Formatter:
+    """Base class for docstring formatter"""
+
+    @abc.abstractmethod
+    def treat_token(self, tokeninfo: tokenize.TokenInfo) -> tokenize.TokenInfo:
+        """Return a modified token"""
```

Comment:
```suggestion
        """Return a modified token."""
```

Reply:
Huhu, you can't wait for this new DotFormatter huh 😄

Reply:
Well, I'm actually a bit hesitant. I would like to do something like "CheckIfSentenceFormatter" first, as adding a dot to non-sentences is weird and might shy users away from using the tool.

Perhaps I can leverage the work you're doing in #15 to make it an optional formatter to begin with 😄

Reply:
Yes it's non trivial, I think a first version could add a line feed after existing dot and add a dot to string not ending with ``:``.